### ISSUE TITLE
Uncaught exception due to missing file

### ISSUE THREAD

**Issue by reservoirinvest** [jupyterlab-contrib/jupyterlab_code_formatter#352]
Uncaught exception GET /jupyterlab_code_formatter

      File "~\python\nse\.venv\Lib\site-packages\tornado\web.py", line 1788, in _execute
        result = method(*self.path_args, **self.path_kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "~\python\nse\.venv\Lib\site-packages\tornado\web.py", line 3301, in wrapper
        return method(self, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "~\python\nse\.venv\Lib\site-packages\jupyterlab_code_formatter\handlers.py", line 18, in get
        "formatters": {
                      ^
      File "~\python\nse\.venv\Lib\site-packages\jupyterlab_code_formatter\handlers.py", line 20, in <dictcomp>
        "enabled": formatter.cached_importable if use_cache else formatter.importable,
                                                                 ^^^^^^^^^^^^^^^^^^^^
      File "~\python\nse\.venv\Lib\site-packages\jupyterlab_code_formatter\formatters.py", line 364, in importable
        package_location = subprocess.run(
                           ^^^^^^^^^^^^^^^
      File "~\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 548, in run
        with Popen(*popenargs, **kwargs) as process:
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "~\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 1026, in __init__
        self._execute_child(args, executable, preexec_fn, close_fds,
      File "~\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 1538, in _execute_child
        hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    FileNotFoundError: [WinError 2] The system cannot find the file specified
[W 2024-07-30 01:55:58.738 ServerApp] wrote error: 'Unhandled error'
    Traceback (most recent call last):
      File "~\python\nse\.venv\Lib\site-packages\tornado\web.py", line 1788, in _execute
        result = method(*self.path_args, **self.path_kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "~\python\nse\.venv\Lib\site-packages\tornado\web.py", line 3301, in wrapper
        return method(self, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "~\python\nse\.venv\Lib\site-packages\jupyterlab_code_formatter\handlers.py", line 18, in get
        "formatters": {
                      ^
      File "~\python\nse\.venv\Lib\site-packages\jupyterlab_code_formatter\handlers.py", line 20, in <dictcomp>
        "enabled": formatter.cached_importable if use_cache else formatter.importable,
                                                                 ^^^^^^^^^^^^^^^^^^^^
      File "~\python\nse\.venv\Lib\site-packages\jupyterlab_code_formatter\formatters.py", line 364, in importable
        package_location = subprocess.run(
                           ^^^^^^^^^^^^^^^
      File "~\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 548, in run
        with Popen(*popenargs, **kwargs) as process:
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "~\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 1026, in __init__
        self._execute_child(args, executable, preexec_fn, close_fds,
      File "~\AppData\Local\Programs\Python\Python311\Lib\subprocess.py", line 1538, in _execute_child
        hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    FileNotFoundError: [WinError 2] The system cannot find the file specified

**Comment by st-bender** [comment]
Hi there,
The same problem on Linux.
But the message is a bit more specific:

```
FileNotFoundError: [Errno 2] No such file or directory: 'Rscript'
```

HTH.

**Comment by ctcjab** [comment]
I'll look out for the next release so I can test it when it's out, and post back here if I see anything unexpected.

**Comment by krassowski** [comment]
@ctcjab and others, can you confirm that this is fixed after installing the latest release [`v3.0.1`](https://github.com/jupyterlab-contrib/jupyterlab_code_formatter/releases/tag/v3.0.1)? It is already on PyPI, will be on conda-forge soon.

**Comment by ctcjab** [comment]
Confirmed, thanks!
